OpenTelemetrySpanTest.transactionRunnerWithFailedAndBeginTransaction is flaky

```
[INFO] Running com.google.cloud.spanner.OpenTelemetrySpanTest
Error:  Tests run: 8, Failures: 1, Errors: 0, Skipped: 0, Time elapsed: 0.563 s <<< FAILURE! -- in com.google.cloud.spanner.OpenTelemetrySpanTest
Error:  com.google.cloud.spanner.OpenTelemetrySpanTest.transactionRunnerWithFailedAndBeginTransaction -- Time elapsed: 0.058 s <<< FAILURE!
java.lang.AssertionError: expected:<[CloudSpanner.ReadWriteTransaction, CloudSpannerOperation.BatchCreateSessions, CloudSpannerOperation.BatchCreateSessionsRequest, CloudSpannerOperation.BeginTransaction, CloudSpannerOperation.Commit, CloudSpannerOperation.CreateMultiplexedSession, CloudSpannerOperation.ExecuteUpdate]> but was:<[CloudSpanner.ReadWriteTransaction, CloudSpannerOperation.BatchCreateSessions, CloudSpannerOperation.BatchCreateSessionsRequest, CloudSpannerOperation.BeginTransaction, CloudSpannerOperation.Commit, CloudSpannerOperation.ExecuteUpdate]>
	at org.junit.Assert.fail(Assert.java:89)
	at org.junit.Assert.failNotEquals(Assert.java:835)
	at org.junit.Assert.assertEquals(Assert.java:120)
	at org.junit.Assert.assertEquals(Assert.java:146)
	at com.google.cloud.spanner.OpenTelemetrySpanTest.verifySpans(OpenTelemetrySpanTest.java:834)
	at com.google.cloud.spanner.OpenTelemetrySpanTest.transactionRunnerWithFailedAndBeginTransaction(OpenTelemetrySpanTest.java:660)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:59)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:56)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.internal.runners.statements.RunBefores.evaluate(RunBefores.java:26)
	at org.junit.internal.runners.statements.RunAfters.evaluate(RunAfters.java:27)
	at org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)
	at org.junit.runners.BlockJUnit4ClassRunner$1.evaluate(BlockJUnit4ClassRunner.java:100)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:366)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:103)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:63)
	at org.junit.runners.ParentRunner$4.run(ParentRunner.java:331)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:79)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:329)
	at org.junit.runners.ParentRunner.access$100(ParentRunner.java:66)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:293)
	at org.junit.internal.runners.statements.RunBefores.evaluate(RunBefores.java:26)
	at org.junit.internal.runners.statements.RunAfters.evaluate(RunAfters.java:27)
	at org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:413)
	at org.junit.runners.Suite.runChild(Suite.java:128)
	at org.junit.runners.Suite.runChild(Suite.java:27)
	at org.junit.runners.ParentRunner$4.run(ParentRunner.java:331)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:79)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:329)
	at org.junit.runners.ParentRunner.access$100(ParentRunner.java:66)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:293)
	at org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:413)
	at org.apache.maven.surefire.junitcore.JUnitCore.run(JUnitCore.java:49)
	at org.apache.maven.surefire.junitcore.JUnitCoreWrapper.createRequestAndRun(JUnitCoreWrapper.java:120)
	at org.apache.maven.surefire.junitcore.JUnitCoreWrapper.executeEager(JUnitCoreWrapper.java:95)
	at org.apache.maven.surefire.junitcore.JUnitCoreWrapper.execute(JUnitCoreWrapper.java:75)
	at org.apache.maven.surefire.junitcore.JUnitCoreWrapper.execute(JUnitCoreWrapper.java:69)
	at org.apache.maven.surefire.junitcore.JUnitCoreProvider.invoke(JUnitCoreProvider.java:146)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:385)
	at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:162)
	at org.apache.maven.surefire.booter.ForkedBooter.run(ForkedBooter.java:507)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:495)
```

--- a/google-cloud-spanner/src/test/java/com/google/cloud/spanner/OpenTelemetrySpanTest.java
+++ b/google-cloud-spanner/src/test/java/com/google/cloud/spanner/OpenTelemetrySpanTest.java
@@ -572,22 +572,13 @@ public class OpenTelemetrySpanTest {
   @Test
   public void transactionRunnerWithFailedAndBeginTransaction() {
     List<String> expectedReadWriteTransactionWithCommitAndBeginTransactionSpans =
-        isMultiplexedSessionsEnabled()
-            ? ImmutableList.of(
-                "CloudSpannerOperation.CreateMultiplexedSession",
-                "CloudSpannerOperation.BeginTransaction",
-                "CloudSpannerOperation.BatchCreateSessionsRequest",
-                "CloudSpannerOperation.ExecuteUpdate",
-                "CloudSpannerOperation.Commit",
-                "CloudSpannerOperation.BatchCreateSessions",
-                "CloudSpanner.ReadWriteTransaction")
-            : ImmutableList.of(
-                "CloudSpannerOperation.BeginTransaction",
-                "CloudSpannerOperation.BatchCreateSessionsRequest",
-                "CloudSpannerOperation.ExecuteUpdate",
-                "CloudSpannerOperation.Commit",
-                "CloudSpannerOperation.BatchCreateSessions",
-                "CloudSpanner.ReadWriteTransaction");
+        ImmutableList.of(
+            "CloudSpannerOperation.BeginTransaction",
+            "CloudSpannerOperation.BatchCreateSessionsRequest",
+            "CloudSpannerOperation.ExecuteUpdate",
+            "CloudSpannerOperation.Commit",
+            "CloudSpannerOperation.BatchCreateSessions",
+            "CloudSpanner.ReadWriteTransaction");
     DatabaseClient client = getClient();
     assertEquals(
         Long.valueOf(1L),
@@ -612,7 +603,7 @@ public class OpenTelemetrySpanTest {
     Stopwatch stopwatch = Stopwatch.createStarted();
     while (spanExporter.getFinishedSpanItems().size()
             < expectedReadWriteTransactionWithCommitAndBeginTransactionSpans.size()
-        && stopwatch.elapsed().compareTo(java.time.Duration.ofMillis(1000)) < 0) {
+        && stopwatch.elapsed(TimeUnit.MILLISECONDS) < 2000) {
       Thread.yield();
     }
 
@@ -621,7 +612,11 @@ public class OpenTelemetrySpanTest {
         .getFinishedSpanItems()
         .forEach(
             spanItem -> {
-              actualSpanItems.add(spanItem.getName());
+              // Ignore multiplexed sessions, as they are not used by this test and can therefore
+              // best be ignored, as it is not 100% certain that it has already been created.
+              if (!"CloudSpannerOperation.CreateMultiplexedSession".equals(spanItem.getName())) {
+                actualSpanItems.add(spanItem.getName());
+              }
               switch (spanItem.getName()) {
                 case "CloudSpannerOperation.CreateMultiplexedSession":
                   verifyRequestEvents(

